tests/integration/SdkTest_test.cpp

```diff
@@ -4696,6 +4704,117 @@ TEST_F(SdkTest, SdkGetBanners)
     ASSERT_TRUE(err == MegaError::API_OK || err == MegaError::API_ENOENT) << "Get banners failed (error: " << err << ")";
 }
 
+TEST_F(SdkTest, SdkBackupFolder)
+{
+    getAccountsForTest(1);
+    LOG_info << "___TEST BackupFolder___";
+
+    // Attempt to get My Backups folder;
+    // make this work even before the remote API has support for My Backups folder
+    synchronousGetMyBackupsFolder(0);
+    MegaHandle mh = mApi[0].h;
+
+    // create My Backups folder
+    unique_ptr<MegaNode> myBkpsNode{ mh && mh != UNDEF ? megaApi[0]->getNodeByHandle(mh)
+                                                       : megaApi[0]->getNodeByPath("/My Backups") };
```

Comment:
The `My Backups` folder should be retrieved exclusively from the `mh` handle, not by node's name.
Also, the `UNDEF` is internal. Here, you should use `INVALID_HANDLE` instead
```suggestion cpp
    unique_ptr<MegaNode> myBkpsNode{ mh && mh != INVALID_HANDLE ? megaApi[0]->getNodeByHandle(mh) : nullptr };
```

Reply:
Actually, taking into account the `myBkpsNode` is only here, I'd say the `if (!myBkpsNode)` could be replaced directly by:
``` cpp
if (unique_ptr<MegaNode> n = megaApi[0]->getNodeByHandle(mh))
```

Reply:
How would this work? It needs only the negated condition.